server/data/testutils/mocks.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/first */
+/*
+ * Import from '..' (server/data/index.ts) fails if applicationInfo not mocked first. This is
```

Comment:
I think this bit of extra complexity is because of a very recent change in the template app. More info: https://github.com/ministryofjustice/hmpps-template-typescript/pull/190#discussion_r1192515934